logging/google/cloud/logging/handlers/_helpers.py

```diff
@@ -37,3 +47,61 @@ def format_stackdriver_json(record, message):
     }
 
     return json.dumps(payload)
+
+
+def get_trace_id_from_flask():
+    """Get trace_id from flask request headers.
+
+    :rtype: str
+    :return: Trace_id in HTTP request headers.
+    """
+    if flask is None or not flask.request:
+        return None
+
+    header = flask.request.headers.get(_FLASK_TRACE_HEADER)
+
+    if header is None:
+        return None
+
+    trace_id = header.split('/')[0]
+
+    return trace_id
+
+
+def get_trace_id_from_django():
+    """Get trace_id from django request headers.
+
+    :rtype: str
+    :return: Trace_id in HTTP request headers.
+    """
+    request = _get_django_request()
+
+    if request is None:
+        return None
+
+    try:
+        header = request.META.get(_DJANGO_TRACE_HEADER)
+    except KeyError:
+        return None
+
+    if header is None:
+        return None
+    trace_id = header.split('/')[0]
```

Comment:
super style nit: above you put a newline above this, but here it's scrunched up against the if statement above. No action needed, but something to think about.